internal/db/prepare_transaction.go

```diff
@@ -0,0 +1,30 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+)
+
+type Order struct {
+	ID        uuid.UUID `db:"id"`
+	UserID    uuid.UUID `db:"user_id"`
+	Label     string    `db:"label"`
+	CreatedAt time.Time `db:"created_at"`
+}
+
+func InsertUser(ctx context.Context, dbConn sqlx.ExecerContext, order *Order) error {
+	query := "INSERT INTO orders ( id,  user_id, label, created_at ) VALUES ($1, $2, $3, $4)"
```

Comment:
try to use named placeholders, not positional.
named parameters are more maintainable.